extensions/SemanticForms/includes/SF_ParserFunctions.php

```diff
@@ -465,7 +465,7 @@ static function renderArrayMapTemplate( &$parser, $frame, $args ) {
 	}
 
 
-	static function renderAutoEdit( &$parser ) {
+	static function renderAutoEdit( $parser ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') The Cyclomatic Complexity of this function "renderAutoEdit" is 23 which is greater than 20 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1541)